File: Ruby/lib/DeathBadConsequence.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

module NapakalakiGame
  
  require_relative 'Player.rb'
  require_relative 'BadConsequence.rb'
  require_relative 'NumericBadConsequence.rb'
  
  class DeathBadConsequence < NumericBadConsequence
    
    attr_accessor :death
    
    def initialize(text)
      super(text,10, @@MAXTREASURES, @@MAXTREASURES)
    end
    
  end

end

File: src/napakalaki/DeathBadConsequence.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalaki;

/**
 *
 * @author Oscar
 */
public class DeathBadConsequence extends NumericBadConsequence{
    // Player player;
    private boolean death;
    
    public DeathBadConsequence(String text, boolean death){
        super(text,10, MAXTREASURES, MAXTREASURES);
        this.death = death;
    }
    
    public boolean getDeath(){
        return death;
    }
}
